### PR TITLE
new devicetype LS-113G

### PR DESCRIPTION
**Pull request by J2thatsme**

### CODE REVIEW

File: types/globalsat/LS-113G/default.schema.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    },
+    "co2": {
+      "title": "CO2",
+      "unit": "ppm",
+      "type": "integer",
+      "description": "Carbon Dioxide in ppm",
+      "minimum": 0,
+      "maximum": 10000
+    },
+    "co": {
```
Comment (mariusgiger) [inline types/globalsat/LS-113G/default.schema.json:31]: change it to co2?

File: types/globalsat/LS-113G/default.schema.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      "title": "CO",
```
Comment (mariusgiger) [inline types/globalsat/LS-113G/default.schema.json:32]: CO2

File: types/globalsat/LS-113G/default.schema.json [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+      "unit": "ppm",
+      "type": "integer",
+      "description": "Carbon Dioxide in ppm",
+      "minimum": 0,
+      "maximum": 10000
+    },
+    "pm2_5": {
+      "title": "Particles 2.5",
```
Comment (mariusgiger) [inline types/globalsat/LS-113G/default.schema.json:40]: change to Particulate Matter?

File: types/globalsat/LS-113G/uplink.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    data.pm2_5 = Bits.bitsToUnsigned(bits.substr(40, 16));
+  }
+
+  data.temperature = Bits.bitsToUnsigned(bits.substr(8, 16)) / 100;
+  data.humidity = Bits.bitsToUnsigned(bits.substr(24, 16)) / 100;
+
+  var sample = { data: data };
+
+  emit('sample', sample);
+
```
Comment (mariusgiger) [inline types/globalsat/LS-113G/uplink.js:23]: remove unused whitespace

**Review by mariusgiger — CHANGES_REQUESTED** [review]

File: types/globalsat/LS-113G/default.schema.json (hunk 1 above)
Comment (J2thatsme) [reply types/globalsat/LS-113G/default.schema.json:31]: image
It's CO ^^

Comment (mariusgiger) [reply types/globalsat/LS-113G/default.schema.json:31]: then the description is wrong

**Review by mariusgiger — APPROVED** [review]

**Pull request merged**